### PR TITLE
Add coconut margarita recipe

### LINKED ISSUE #882: Add a drink! 🍸
Add your drink recipes by forking and submitting a pull request.

Be sure to check out the [Contributing Guidelines ](https://github.com/alfg/opendrinks/blob/master/CONTRIBUTING.md )before submitting a pull request. :)

Happy hacking!

### PR DESCRIPTION
**Pull request by ferdox2**
fix #882 
Adds coconut margarita recipe.

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Image is stretched. Please provide an image 600x400px in size while maintaining the aspect ratio. Thanks!

**Comment by ferdox2** [comment]
done

File: src/recipes/coconut-margarita.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            "ingredient": "Triple Sec"
+        },
+        {
+            "quantity": "1/4",
+            "measure": "Cup",
+            "ingredient": "Sweetened flaked coconut"
+        }
+    ],
+    "directions": [
+        "Fill a blender with ice; add the coconut cream, tequila, and triple sec; blend at high speed until smooth. Scatter the coconut onto a plate. Wet the rims of four glasses. Dip the glass rims in the coconut. Serve the margaritas in the prepared glasses."
```
Comment (alfg) [inline src/recipes/coconut-margarita.json:33]: Can you split these up into multiple steps?

Comment (ferdox2) [reply src/recipes/coconut-margarita.json]: done

**Comment by alfg** [comment]
Thanks!

**Pull request merged**